extensions/ql-vscode/src/vscode-tests/no-workspace/databaseFetcher.test.ts

```diff
@@ -13,11 +13,13 @@ import {
   looksLikeLgtmUrl,
   findDirWithFile,
 } from '../../databaseFetcher';
+import { ProgressUpdate } from '../../commandRunner';
 chai.use(chaiAsPromised);
 const expect = chai.expect;
 
-describe('databaseFetcher', function() {
+describe('databaseFetcher', function () {
   // These tests make API calls and may need extra time to complete.
+  const fakeProgress = (_: ProgressUpdate) => { /*do nothing*/ };
```

Comment:
This could be converted into a test spy and you can later verify that this spy is invoked. This would ensure that the progress update is properly called.

eg- 

```
let progressSpy: sinon.SinonStub<ProgressCallback>;

// and then in the before block:
progressSpy = sandbox.spy();

// and then in each test, after the call to convert databasea
expect(progressSpy).to.have.been.calledOnce;